### PR TITLE
Add draft release notes for first RC of 1.2.0.

### PR DESCRIPTION
**Pull request by mike-jumper**

### CODE REVIEW

File: _releases/1.2.0.md [hunk 1]
```diff
@@ -0,0 +1,405 @@
...
+ * [GUACAMOLE-518](https://issues.apache.org/jira/browse/GUACAMOLE-518) - Shift incorrectly sent for uppercase letters while Caps Lock is active
+ * [GUACAMOLE-859](https://issues.apache.org/jira/browse/GUACAMOLE-859) - Incorrect Caps Lock keysym sent to Windows via RDP
+ * [GUACAMOLE-917](https://issues.apache.org/jira/browse/GUACAMOLE-917) - Key mapping for German keyboards: ~ does not work
+
+### VNC support for TLS
+
+While Guacamole will automatically use TLS for VNC servers that require it, the
+proper locking callbacks required for multithreaded use of TLS were not set,
+resulting in unstable behavior. These callbacks are now correctly specified,
+and connections to VNC servers requiring TLS should function correctly.
```
Comment (necouchman) [inline _releases/1.2.0.md:310]: Not sure if it needs to be noted in these release notes or not, but this is also dependent upon the correct version of `libvncserver`/`libvncclient` on the system running guacd, as that library also has to support the locking.

Comment (mike-jumper) [reply _releases/1.2.0.md:310]: Sure, I'll clarify that.

Comment (necouchman) [reply _releases/1.2.0.md:310]: No worries, I wasn't 100% sure it actually belongs in these release notes, but it seems like it possibly may stave off a few errant JIRA issues entered when people claim that it isn't working correctly but haven't examined their libvnc version.  Maybe.  Or, it'll at least give us a place to point when they do...

**Comment by mike-jumper** [comment]
Aaaallll set.

File: _releases/1.2.0.md [hunk 2]
```diff
@@ -0,0 +1,410 @@
...
+
+Guacamole now supports the MariaDB version of "Connector/J" in addition to the
+MySQL version.
+
+ * [GUACAMOLE-852](https://issues.apache.org/jira/browse/GUACAMOLE-852) - Support MariaDB JDBC Driver
+
+### Support for SQL Server instance names
+
+SQL Server supports deploying multiple instances on the same server, with each
+additional instance identified its own unique name. If using SQL Server, and
```
[the remark this answers is not in the document]
Comment (mike-jumper) [reply _releases/1.2.0.md]: OK - fixed and rebased.

**Pull request merged**